### ISSUE TITLE
Offline or inactive command bursts show incorrect burst strength in Miscellanea column

### ISSUE THREAD

**Issue by shyadow** [pyfa-org/Pyfa#2393]
## Bug Report
Command Bursts show a strength in the Miscellanea column even when they are not active or offline.
![image](https://user-images.githubusercontent.com/91671235/146844988-92fe4c17-8f88-446b-a944-6cd4d330392b.png)

### Expected behavior:
Command Burst strength should be zero when the command burst module is offline or not active.

### Actual behavior:
Command Bursts show a small value when turned off or offline (see screenshot).

### Detailed steps to reproduce:
1. Open a fit with command bursts
2. Add charges to the command burst
3. Offline or set command burst to not active

### Fits involved in EFT format (Edit > To Clipboard > EFT):
```
[Absolution, Max Armor Links]

[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]

[Empty Med slot]
[Empty Med slot]
[Empty Med slot]

Armor Command Burst II, Armor Energizing Charge /OFFLINE
Armor Command Burst II, Armor Reinforcement Charge
Armor Command Burst II, Rapid Repair Charge
[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]

Medium Command Processor I
[Empty Rig slot]


Armored Command Mindlink
```

### Release or development git branch? Please note the release version or commit hash:
v2.39.1 (probably introduced in v2.39.0dev2 with the addition of showing command burst strength in the Miscellanea column though)

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
Windows 10

### Other relevant information:
This doesn't affect the actual stats of the ship (e.g. the ship is not getting a 1.5% Armor HP bonus), it's just the information in the Miscellanea column.

**Comment by shyadow** [comment]
Burst strength doesn't necessarily have to be zero when not active/offline, it could also be full strength (more consistent with behavior of other modules), but in any case it shouldn't be a different value, like 1.5% in this case.

**Comment by DarkFenX** [comment]
Fixed in 66018642f75ecf41efed51de0cd5a24a96fb6d01
